### PR TITLE
import relationships, increase test coverage, rearrange wire package

### PR DESCRIPTION
**Pull request by dwhitena**
# What this PR does:
- adds relationship graph importing/removing functionality to the wire API
- increasing wire test coverage
- rearranges code within wire to read from top to bottom
- updates view test fixture.

# How to test this PR:
- `go test ./...` from `internal`

### CODE REVIEW

File: internal/wire/wire.go [hunk 1]
```diff
@@ -89,3 +108,227 @@ func Execute(context interface{}, mgoDB *db.DB, graphDB *cayley.Handle, viewPara
...
+			err := fmt.Errorf("Invalid view path level, expected %d but seeing %d", level, segment.Level)
+			return graphPath, err
+		}
+
+		// Initialize the path, if we are on level 1.
+		if level == 1 {
+
+			// Add the first level relationship.
+			switch segment.Direction {
+			case "in":
```
Comment (wyattjoh) [inline internal/wire/wire.go]: Would it not make sense to move the strings `in` and `out` into package const's?

Comment (dwhitena) [inline internal/wire/wire.go]: Good idea.  I will make the change.

File: internal/wire/wire_test.go [hunk 2]
```diff
@@ -320,3 +372,238 @@ func TestPersistView(t *testing.T) {
...
+		t.Fatalf("\t%s\tShould be able to get a Mongo session : %v", tests.Failed, err)
+	}
+	defer db.CloseMGO(tests.Context)
+
+	opts := make(map[string]interface{})
+	opts["database_name"] = cfg.MustString("MONGO_DB")
+	opts["username"] = cfg.MustString("MONGO_USER")
+	opts["password"] = cfg.MustString("MONGO_PASS")
+	store, err := cayley.NewGraph("mongo", cfg.MustString("MONGO_HOST"), opts)
+	if err != nil {
```
Comment (wyattjoh) [inline internal/wire/wire_test.go]: Possible to extract the database setup into a `setup` function as was done for xenia? Ref: https://github.com/coralproject/shelf/pull/192/commits/adae938946d25a22f954a2c2a978c483e8bd4f4f

**Comment by dwhitena** [comment]
Need to follow up on the DB setup/teardown in tests: https://github.com/coralproject/shelf/issues/195

**Pull request merged**